CMakeUtilities/InstallCMakeExports.cmake

```diff
@@ -111,12 +113,32 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/cmake/FindSIMDIS_SDK.cmake
     DESTINATION ${SDK_LIB_CMAKE_PATH}
     COMPONENT Exports
 )
+endif()
+
+if(USING_VCPKG)
+# Configure and install the ImportSIMDIS_SDK.cmake useful for external projects
+configure_file(CMakeUtilities/FindSIMDIS_SDK.cmake.in
```

Comment:
Please fix spacing in CMake to abide by code style. 4 spaces indent. I did a pass fixing already, and an update has added them back in. PR will not be accepted without abiding by existing style.